src/modules/sensors/voted_sensors_update.cpp

```diff
@@ -730,7 +659,6 @@ void VotedSensorsUpdate::gyro_poll(struct sensor_combined_s &raw)
 		if (_gyro.last_best_vote != best_index) {
 			_gyro.last_best_vote = (uint8_t)best_index;
 			_corrections.selected_gyro_instance = (uint8_t)best_index;
```

Comment:
This is wrong.

Reply:
The `selected_gyro_instance` needs to be published.

Reply:
Hmm... I think we need to split these topics
I'd like to rename `sensor_correction` to `sensor_thermal_cal` and move all of
```
	uint8_t selected_gyro_instance;
	uint8_t selected_accel_instance;
	uint8_t selected_baro_instance;
	uint8_t gyro_mapping[3];
	uint8_t accel_mapping[3];
	uint8_t baro_mapping[3];
```
these into their own topic. Does this make sense?

Reply:
I mean... we kinda have to with this change. TemperatureCompensationModule has no business knowing/setting the `selected_*_instance` or the `*_mapping`

Reply:
I believe `sensor_correction` could move entirely to temperature_compensation and the selected instance (and mapping) could go away entirely.

Then in the sensors module `sensor_selection` could also contain the ORB instance number.